examples/first.go

```diff
@@ -3,13 +3,14 @@ package main
 import (
 	"fmt"
 
-	"github.com/caarlos0/env"
```

Comment:
I switched it while writing the patch (since I was working in my fork and tests wouldn't pass otherwise). I can push a quick update to both.